applications/accounting/src/main/groovy/org/apache/ofbiz/admin/AcctgAdminServices.groovy

```diff
@@ -15,7 +15,10 @@
  * KIND, either express or implied.  See the License for the
  * specific language governing permissions and limitations
  * under the License.
- */
+*/
+
+package org.apache.ofbiz.admin
```

Comment:
Package name should be org.apache.ofbiz.accounting.admin, Like we have package hierarchy for java.